ui/package.json

```diff
@@ -5,27 +5,30 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build --mode production --target lib --report --name FundamentalVue src/main.ts",
+    "test": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "test": "vue-cli-service test:unit \"src/**/__tests__/*.test.{ts,js,vue}\"",
+    "test:unit": "vue-cli-service test:unit",
```

Comment:
Same as test. Either remove, or point to test.

Reply:
removed :)